### PR TITLE
#CP-14 Feat: Simulate bus motions

### LINKED ISSUE #25: Fix inconsistency with how API responses are being sent
There is inconsistency with how responses are now being sent from the backend especially when we are sending just short  messages to the user 
the `handleResponse` function that we are using to format our responses returns a response that includes `status`, `code`, and `data`, and this data field that actually contains information from the backend should be an object itself and it is in most cases but there are some cases the message is being sent just as a string especially when we are returning just a short message like `user deleted`, `operator updated`...

these changes should be made

```diff
{
status: "success",
code: "200",
- data: "driver updated successfully"
+ data: {message: "driver updated successfully"}
}
```
making our API responses consistent will prevent unnecessary checks that can be made during integration when someone doesn't know the exact data structure of the message coming from the backend

### PR DESCRIPTION
**Pull request by feyton**
### What does this PR do?
As **a driver**, I want to be able to command the bus to start, stop, speed up, and slow down the bus so that the bus can move.
### Description of Task to be 

- [x] The driver should be able to command the bus to start, move and stop.
- [x] The driver should be able to command the bus to speed up and slow down.
- [x] The driver should be able to update bus info on bus stops. e.g. number of passengers alighted and boarded.
- [x] The users should be able to view the bus movement in real-time.
- [x] Appropriate error messages are displayed on unsuccessful attempts

### How should this be manually tested?

> Docker instructions to be added with corresponding image

### CODE REVIEW

File: src/simulateApp/validation.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { body } from 'express-validator';
+
+export const validatePassengers = () => [
+	body('passengers', 'This should be a number').optional().toInt(),
+];
```
Comment (Abdulkeza) [inline src/simulateApp/validation.js]: What will happen if a driver entered like 1 million as the total seats a car can hold? 
or Zero seats???
ie if you can just add validations for that...

**Review by Abdulkeza — COMMENTED** [review]
You did a great job, Mr. Fabrice, this is incredible.. so left some comments bellow

File: src/simulateApp/validation.js (hunk 1 above)
Comment (feyton) [reply src/simulateApp/validation.js]: Thank you for mentioning @Abdulkeza 
I have added the validation to ensure that we are not exceeding the number of seats

**Review by samyshyaka — DISMISSED** [review]
This is a great implementation of the feature

**Review by Abdulkeza — DISMISSED** [review]
Great Job

**Review by Abdulkeza — DISMISSED** [review]
Great job

**Review by samyshyaka — DISMISSED** [review]

**Review by Jkadhuwa — APPROVED** [review]

**Review by samyshyaka — APPROVED** [review]

**Review by Abdulkeza — APPROVED** [review]
good job

**Pull request merged**